db/models.py

```diff
@@ -1 +1,14 @@
 from django.db import models
+
```

Comment:
There is an unnecessary import from `symtable` that is not used in this file. It's a good practice to remove unused imports to keep the code clean and maintainable.